RELEASE/scripts/autoscend/iotms/mr2017.ash

```diff
@@ -1698,6 +1698,11 @@ boolean horsePreAdventure()
 	return getHorse(desiredHorse);
 }
 
+boolean auto_haveGenieBottle()
+{
+	return (item_amount($item[Genie Bottle]) > 0 && auto_is_valid($item[Genie Bottle]));
```

Comment:
Should this also check if we have pocket wishes and they are valid?

I'm not sure if functionally different since have to use the bottle to generate pocket wishes. Maybe someone manually pulls a pocket wish? Full disclaimer - I don't have genie so my understanding could very well be wrong.

Reply:
nah, you are right. if the user has pocket wishes, they can wish for effects. a better way to set this up might be like... haveGenieBottleOrPocketWishes or something so it is more explicit that it is checking for both, but it really should check for both

Reply:
I'm fine with this being done, but it should probably be written/tested by someone with a genie bottle rather than me.

Reply:
Well, I've written and committed it but it's pretty much untestd as I don't have a bottle.